.gitignore

```
@@ -0,0 +1,2 @@
+debug
+target
```

Does `debug` refer to the `target/debug` directory? Typically, only the `target/` directory is listed in the ignore file.
```suggestion
target/
```